src/app/features/home/capture-tab/capture-details/capture-details.page.ts

```diff
@@ -141,6 +144,22 @@ export class CaptureDetailsPage {
       .subscribe();
   }
 
+  openCertificate() {
+    zip(this.proof$, this.diaBackendAuthService.token$)
+      .pipe(
+        concatMap(([proof, token]) =>
+          iif(
+            () => proof.diaBackendAssetId !== undefined,
+            Browser.open({
```

Comment:
Wrap the promise with `defer()` so that the promise won't be created immediately when the `iif` is subscribed, causing the Browser to open even when `proof.diaBackendAssetId` is `undefined`.